### DISCUSSION TITLE
Model disappears during staging & camera change on iOS 15.2 only (since v2.0.0)

### DISCUSSION THREAD

**Discussion by Stynson** [google/model-viewer#3876] [Q&A]
### Description

After updating to v2.0.0 on iOS the model disappears during staging & camera change (rotation/zoom/pan etc.)
It reappears after the interaction finished. This includes user interactions and also programatically initiated ones.
The issue 100% reproducible with all modelviewer.dev examples too on iPhones (with iOS 15.2)

https://user-images.githubusercontent.com/19240196/193558721-3a748cc6-6c60-435d-80f7-f058c61e3941.MP4

Same issue happens with animated models too without user interaction:

https://user-images.githubusercontent.com/19240196/193560967-c5566e92-272e-4796-b951-85551cf0e926.MP4

Reproduce: 
Visit modelviewer.dev/examples with an iPhone running iOS 15 and rotate the models.

Expected behavior: 
Model should be visible during the rotation.

#### Live Demo
visit this with iPhone running iOS 15.2:
https://modelviewer.dev/examples/stagingandcameras/

### Version
- model-viewer: v2.0.0

### Browser Affected
- [x] Chrome
- [ ] Edge
- [x] Firefox
- [ ] IE
- [x] Safari

### OS
- [ ] Android
- [x] iOS, version: 15.2  ( it can't be reproduced with iOS 16.0 or iOS 15.4.1 )
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
Yeah, the iOS 15 rollout was a bit of a WebGL disaster. I'm not super excited to try to work around temporary WebKit bugs, but are you saying v1.12 *was* working properly on iOS 15.2? @grorg any idea what WebKit change this might have been related to?

**Comment by Stynson** [comment]
Indeed v1.12 works properly and v2.0 not that's the interesting part.

**Comment by westcode** [comment]
I can confirm we also started observing this exact issue after upgrading v1.12.1 to v2.0.2.

**Comment by elalish** [comment]
Considering iOS 15.2 didn't even make it onto this version share [chart](https://gs.statcounter.com/ios-version-market-share/), I'm not going to prioritize fixing this, especially since I don't have an iOS 15.2 device to repro with. If any of you would like to debug and submit a PR, I'd be stoked though!

**Reply by Stynson** [reply]
Managed to debug it and opened a PR with the fix: #3941
